src/Compilers/CSharp/Test/CommandLine/CommandLineTestBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests
+{
+    public abstract class CommandLineTestBase : CSharpTestBase
+    {
+        public string WorkingDirectory { get; }
```

Comment:
>public string WorkingDirectory { get; } [](start = 8, length = 39)

Consider changing to `protected string _baseDirectory;` to reduce the number of changes in the PR.

Reply:
That violates our style guidelines 😦